zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -1217,7 +1217,7 @@ where
                 // Separate the funding streams into deferred and non-deferred streams
                 .partition(|(receiver, _)| matches!(receiver, FundingStreamReceiver::Deferred));
 
-        let is_nu6 = NetworkUpgrade::current(&network, height) == NetworkUpgrade::Nu6;
+        let is_nu6 = NetworkUpgrade::current(&network, height) >= NetworkUpgrade::Nu6;
```

Comment:
###### non-blocking

We should rename `is_nu6` to `is_post_nu6` to follow the new logic.

Reply:
Fixed in https://github.com/ZcashFoundation/zebra/pull/9256/commits/e2b4756f76b81eac4a5a88c7921107c03ccf1a38